src/components/ModalAddressQRCode.js

```diff
@@ -90,14 +85,14 @@ class ModalAddressQRCode extends React.Component {
               </button>
             </div>
             <div className="modal-body d-flex flex-column align-items-center">
-              <QRCode onClick={this.openQrCode} size={200} value={`hathor:${this.props.lastSharedAddress}`} />
+              <QRCode size={200} value={`hathor:${this.props.lastSharedAddress}`} />
               <span ref="address" className="mt-3">
                 {this.props.lastSharedAddress}
                 <CopyToClipboard text={this.props.lastSharedAddress} onCopy={this.copied}>
                   <i className="fa fa-clone pointer ml-1" title="Copy to clipboard"></i>
                 </CopyToClipboard>
               </span> 
-              <a href={this.getHref()} download={`Hathor address - ${this.props.lastSharedAddress}`} className="hidden" ref="downloadLink">Download</a>
+              <a href="true" download={`Hathor address - ${this.props.lastSharedAddress}`} className="hidden" ref="downloadLink">Download</a>
```

Comment:
Why `"true"`?

Reply:
The linter requires always a value for `href` when using `<a>`, that's why I use `true` for all `<a>` because it has no side effect